### ISSUE TITLE
Cannot build with OpenSSL 1.1.1c

### ISSUE THREAD

**Issue by phil-brad** [yhirose/cpp-httplib#208]
When building with a CMake generated Visual studio solution (vc140), building against OpenSSL 1.1.1c, the compiler reports that the identifiers `SSL_load_error_strings`, `SSL_library_init` and `ERR_free_strings` are not found. Is this some sort of version issue, or does it not support 1.1.1c yet?

**Comment by yhirose** [comment]
@phil-brad, thanks for the report. I am actually using 1.1.c on Mac OS (installed by homebrew) and there is no problem to build. Could you make sure that your include and library paths are correct?

**Comment by PixlRainbow** [comment]
![image](https://user-images.githubusercontent.com/44422178/63142864-34dd9700-c01e-11e9-8008-ce929e12d1fb.png)
I have run make test using OpenSSL 1.1.1c on an Ubuntu 18.04 docker image. OpenSSL 1.1.1c was built from source as it is not available in default apt-get repositories for Ubuntu 18.04. All tests passed.

**Comment by yhirose** [comment]
@PixlRainbow, thanks for another confirmation!

**Comment by phil-brad** [comment]
I used [FireDaemon's distribution of OpenSSL for Windows](https://kb.firedaemon.com/support/solutions/articles/4000121705), which it turns out is not quite clear what version is built on as in some places it says -c and some places -d (might be straight from repo rather than official release). I used a different distribution of OpenSSL 1.1.1c, and my problems  seem to have been solved. Thanks for your help.
